pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -110,13 +110,25 @@ class Build {
 
     def runTests() {
         def testStages = [:]
-        List testList = []
-
+        List testList = buildConfig.TEST_LIST
+        def java_to_build = buildConfig.JAVA_TO_BUILD
+        jdk_branch = 'dev'
```

Comment:
could jdk_branch be declared as a def or static type?

Also `dev` is the Adopt default branch, I think we should explicitly call that out

Reply:
Updated.